config/settings.yml

```
@@ -0,0 +1,3 @@
+user:
+  max_length_255: 255
+  min_length_6: 6
```

nên đổi lại để có khả năng tái sử dụng cao hơn thay vì truyền model thế này 

```suggestion
validations:
  length:
    digit_255: 255
    digit_6: 6
```